docassemble/AssemblyLine/data/static/al_audio.js

```diff
@@ -1,74 +1,98 @@
+// Do not overwrite already created al_js object. This may be relevant as our js gets built out.
+if ( !al_js ) { var al_js = {}; }
+
 $(document).on('daPageLoad', function() {
-  var $audio_nodes = $('.daaudio-control');
-  var id_count = 1;
-  for ( var audio_node of $audio_nodes ) {
-    al_js.replace_with_audio_minimal_controls( audio_node, 'page_reader_substitute_' + id_count );
-    id_count++;
-  }
+  try {
+    var $audio_nodes = $('.daaudio-control');
+    var id_count = 1;
+    for ( var audio_node of $audio_nodes ) {
+      al_js.replace_with_audio_minimal_controls( audio_node, 'page_reader_substitute_' + id_count, $audio_nodes );
+      id_count++;
+    }
+  } catch ( error ) { console.log( 'AL audio intantiation error:', error ) }
 });
 
-var al_js = {};
-
 // We are not providing a way to rewind or scan through the audio.
 // TODO: Show user the audio still needs to load:
 // - https://stackoverflow.com/questions/9337300/html5-audio-load-event,
 // - https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play
 al_js.replace_with_audio_minimal_controls = function( audio_node, id ) {
   /* Create the custom controls for this specific audio element
-  * and give the new controls container the requested `id` tag. */
+  * and give the new controls container the requested `id` tag.
+  * Return functions to control audio playback for this node. */
+  var controls = {
+    play: function () {}, pause: function () {},
+    restart: function () {}, stop_and_reset: function () {},
+  };
+  
   if (!audio_node) {
     // Some screens, like the signature screen, don't have screen readers.
-    return;
+    return controls;
```

Comment:
I'm confused about this because you're returning the controls object, but when you call the function on line 9, you're not doing anything with returned obj?

Reply:
It's true, I'm not doing so right now. I did see the potential for it's use during development, though. It's a bit of premature optimization and if we want to remove it we can.